ted2zim/scraper.py

```diff
@@ -531,7 +531,9 @@ def extract_video_info_from_json(self, json_data):
         video_id = talk_info["id"]
         speaker_profession = speaker_info["description"]
         speaker_bio = speaker_info["whotheyare"]
-        speaker_picture = speaker_info["photo_url"]
+        speaker_picture = (
```

Comment:
with change above, previous version would be empty string and thus evaluate to False anyway…

Reply:
Oh. Yup!